static/src/javascripts/projects/commercial/modules/third-party-tags.ts

```diff
@@ -16,7 +17,7 @@ import { commercialFeatures } from '../../common/modules/commercial/commercial-f
 import { imrWorldwide } from './third-party-tags/imr-worldwide';
 import { imrWorldwideLegacy } from './third-party-tags/imr-worldwide-legacy';
 
-const addScripts = (tags) => {
+const addScripts = (tags: ThirdPartyTag[]) => {
```

Comment:
Locally, the type of `ThirdPartyTag.name` is `any` despite the fact that in `@guardian/commercial-core/dist/cjs/types.d.ts` its type is `VendorName`.
Frontend installs cmp v6.11 which doesn't export `VendorName` whereas commercial-core is using v10.1 which does.
Since the cmp is a dev dependency for commercial-core, v10.1 isn't installed by frontend and so frontend doesn't know about `VendorName`.
I think the options are 
1. Change commercial-core so that consent-management-platform is a fully-fledged dependency rather than a dev dependency
2. bump frontend's to consent-management-platform to version 10.1

Reply:
I vote `2`, but appreciate it might need to happen in another PR. Now that we have a clean v10, let’s keep things up-to-date!

The way of doing `1` is actually to make it a peer dependency, so that `frontend` knows it should have v10, at the very least.

Reply:
Here's a PR for option `2`: https://github.com/guardian/frontend/pull/24511. It seems like releasing those changes before these would solve the problem for the time being.
Maybe we should consider options `1` as well though; if we just upgrade frontend's cmp, we risk the same illusive bug recurring if future versions of the cmp if we make any changes to `VendorName`.